app/messaging/process-data-message.js

```diff
@@ -1,34 +1,42 @@
-const { cacheConfig, messageConfig } = require('../config')
-const { getCachedResponse, setCachedResponse } = require('../cache')
-const sendMessage = require('./send-message')
 const util = require('util')
+const { cacheConfig, messageConfig } = require('../config')
+const { getCachedResponse, setCachedResponse, getCacheKey } = require('../cache')
+const { sendMessage } = require('./send-message')
 const { getData } = require('../data')
+const { TYPE } = require('../constants/type')
+const { validateMessage } = require('./validate-message')
+const { VALIDATION } = require('../constants/errors')
```

Comment:
I wonder if there's a better way to organise these? Though it might be hard with the grouping of { cacheConfig, messageConfig } etc.

Reply:
Yeah, but I think I can be a bit smarter.  Reworked a bit.